les/api_backend.go

```diff
@@ -177,6 +177,14 @@ func (b *LesApiBackend) SendTx(ctx context.Context, signedTx *types.Transaction)
 	return b.eth.txPool.Add(ctx, signedTx)
 }
 
+func (b *LesApiBackend) SendTxs(ctx context.Context, signedTxs []*types.Transaction) []error {
+	return b.eth.txPool.AddBatch(ctx, signedTxs)
+}
+
+func (b *LesApiBackend) SetTimestamp(timestamp int64) {
+	// Intentionally empty because this is not needed for LightChain
```

Comment:
What is a LightChain?

Reply:
Unofficially: no idea

Officially:
> // LightChain represents a canonical chain that by default only handles block
// headers, downloading block bodies and receipts on demand through an ODR
// interface. It only does header validation during chain insertion.